src/osd/osd_types.cc

```diff
@@ -5878,3 +5878,33 @@ ostream& operator<<(ostream& out, const store_statfs_t &s)
       << ")";
   return out;
 }
+
+void OSDOp::clear_data(vector<OSDOp>& ops)
+{
+  for (unsigned i = 0; i < ops.size(); i++) {
```

Comment:
Do you mean having non-static member method of OSDOp, and loop it by caller?
I just feel that it might be cleaner to implement OSDOp related function in OSDOp, since there should be no caller want to clear just one OSDOp data.